stable/yugabyte/templates/service.yaml

```
@@ -637,6 +646,10 @@ spec:
               sleep 60;
               /home/yugabyte/tools/k8s_ybc_parent.py status || /home/yugabyte/tools/k8s_ybc_parent.py start;
             done
+        ports:
+        {{- $ybcPort := index $service.ports "grpc-ybc-port" }}
+          - containerPort: {{ $ybcPort }}
+            name: "grpc-ybc-port"
```

```suggestion
        {{- with index $service.ports "grpc-ybc-port" }}
        ports:
          - containerPort: {{ . }}
            name: "grpc-ybc-port"
        {{- end }}
```

We should use `with` here, as some of the users might have modified the `Services:` section. This section is an array which means these users will have whole section copied and added to the provider overrides. If we don't use with, their deployments will break. They won't have this new grpc-ybc-port field in their overrides.

This port is already exposed in case of normal users who use headless services. So, let's try not to break the installation for them.